oci/archive/oci_src_test.go

```diff
@@ -1,5 +1,26 @@
 package archive
 
-import "github.com/containers/image/v5/internal/private"
+import (
+	"errors"
+	"path/filepath"
+	"testing"
+
+	"github.com/containers/image/v5/internal/private"
+	"github.com/containers/image/v5/types"
+	"github.com/stretchr/testify/assert"
+)
 
 var _ private.ImageSource = (*ociArchiveImageSource)(nil)
+
+func TestNewImageSourceNotFound(t *testing.T) {
+	emptyDir := t.TempDir()
+	sysctx := types.SystemContext{BlobInfoCacheDir: emptyDir}
```

Comment:
(This is not used for anything in this test.)

Reply:
It's needed for `LoadManifestDescriptorWithContext` right?

Reply:
I don’t see where; BIC really only applies to work with _layers_.

Do you mean `BigFilesTemporaryDir`?

Reply:
Oh, you meant `BlobInfoCacheDir` and not `sysctx`.  Sorry, fixed!